resources/sass/app.scss

```diff
@@ -381,6 +381,10 @@ dl.import-meta .download-csv {
     }
 }
 
+.heading-button, .random-mismatches {
```

Comment:
you should add the `heading-button`class  to `.random-mismatches` in `Home.vue` and have a single selector here. or is random-mismatches not a heading button? 

```
<cdx-button
          class="random-mismatches heading-button"
          weight="normal"
          @click="showRandom()"
          :disabled="loading"
        >
```

Reply:
This works for me: 
![image](https://github.com/wmde/wikidata-mismatch-finder/assets/3005239/a5f3180e-aafc-409a-a3f4-b6e859c90311)

Reply:
no, I think you're right. I just wasn't sure how to add it